.github/workflows/ohos.yml

```diff
@@ -52,12 +52,27 @@ jobs:
         run: sudo apt update && python3 ./mach bootstrap --skip-lints
       - name: Setup OpenHarmony SDK
         id: setup_sdk
-        uses: openharmony-rs/setup-ohos-sdk@v0.1
+        uses: openharmony-rs/setup-ohos-sdk@v0.1 # TODO: this needs to be updated
         with:
           version: "4.1"
+          components: "native;toolchains;ets"
+          cache: false
+          fixup-path: true
+      - name: Install node for hvigor
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Install hvigor modules
+        run: |
+          mkdir ~/hvigor-installation
+          cd ~/hvigor-installation
+          echo "@ohos:registry=https://repo.harmonyos.com/npm/" > .npmrc
+          npm install "@ohos/hvigor@4" "@ohos/hvigor-ohos-plugin@4"
+          echo "HVIGOR_PATH=$PWD" >> $GITHUB_ENV
       - name: Build (arch ${{ matrix.arch }} profile ${{ inputs.profile }})
         env:
           OHOS_SDK_NATIVE: ${{ steps.setup_sdk.outputs.ohos_sdk_native }}
+          OHOS_BASE_SDK_HOME: ${{ steps.setup_sdk.outputs.ohos_sdk_native }}
```

Comment:
Is there something that expects this variable name instead of `OHOS_SDK_NATIVE`?

Reply:
hvigor needs this to find the other components like toolchain and ets that it needs for building the shell. The value should actually come from the `ohos_base_sdk_home` output of the setup_sdk step. I'll fix this typo.

Reply:
Interesting. Is this perhaps only needed for hvigor 4.x? I don't think I'm setting this variable when building on the commandline locally.

Reply:
Just tried locally with hvigor 5.x:
```
Info: The OpenHarmony SDK 4.1.7.5 is targeting API-level 11
```
> hvigor ERROR: Unable to find 'sdk.dir' in 'local.properties' or 'OHOS_BASE_SDK_HOME' in the system environment path.
	 at /home/mukilan/servo-ohos/target/openharmony/aarch64-unknown-linux-ohos/release/local.properties
> hvigor ERROR: BUILD FAILED in 1 s 797 ms 
```

Do perhaps have sdk.dir in local.properties or DEVECO_SDK_HOME set?